staging/src/k8s.io/apiserver/pkg/server/healthz.go

```diff
@@ -56,9 +54,9 @@ func (s *GenericAPIServer) installHealthz() {
 
 // installReadyz creates the readyz endpoint for this server.
 func (s *GenericAPIServer) installReadyz(stopCh <-chan struct{}) {
-	s.AddReadyzChecks(shutdownCheck{stopCh})
 	s.readyzLock.Lock()
 	defer s.readyzLock.Unlock()
+	s.AddReadyzChecks(shutdownCheck{stopCh})
```

Comment:
Should we ignore the returned error by `AddReadyzChecks`?

Reply:
I kept the error handling the same as previous.